aggregator/src/helpers/Rng.ts

```diff
@@ -1,18 +1,14 @@
-import { BigNumber, ethers } from "../../deps.ts";
+import { BigNumber } from "../../deps.ts";
 
 import words from "./words.ts";
 
-const { keccak256 } = ethers.utils;
-
 function byteToHex(byte: number): string {
   return byte.toString(16).padStart(2, "0");
 }
 
 function hash(data: string) {
   const byteValues = Array.from(new TextEncoder().encode(data));
-  const hex = `0x${byteValues.map(byteToHex).join("")}`;
-
-  return keccak256(hex);
```

Comment:
This was causing our favorite [err _wrapInput](https://github.com/thehubbleproject/hubble-bls/pull/20) error in tests when generating a random private key. Note that the way it is setup now closely mirrors how the extension generates its private keys.

Reply:
https://github.com/thehubbleproject/hubble-bls/issues/27 , investigating.

Reply:
Fixed with `@thehubbleproject/bls@0.5.1`, reverted this change back to original code.